frontend/src/general/dashboard/DashboardButton.jsx

```diff
@@ -1,10 +1,11 @@
 import React from "react";
 import PropTypes from "prop-types";
 import "./dashboard.css";
-import { Link, Button, withStyles } from "@material-ui/core";
+import { NavLink } from "react-router-dom";
+import { Button, withStyles } from "@material-ui/core";
 
 export default function DashboardButton(props) {
-  const { href, children } = props;
+  const { href, children, selected } = props;
```

Comment:
Rename this to `passedState` or something since we may want to use this in the future to pass some other kind of state.

Reply:
a758f1d